tests/.test.env

```diff
@@ -7,4 +7,4 @@ DAILY_READ_REPORTS_LOCATION='tests/test_reports_location'
 DAILY_READ_FETCH_FROM_NGIS = true
 DAILY_READ_FETCH_FROM_SNPSEQ = true
 DAILY_READ_FETCH_FROM_UGC = true
-#DAILY_READ_LOG_LOCATION='tests/log_location'
+DAILY_READ_USERS_LIST_LOCATION=
```

Comment:
Ok, replaced a  comment with a likely typo? Shouldn't there be something on the right side of the equal sign?

Reply:
Ah no, its not a typo. The field can be empty and should be empty if you want DailyRead to look at all users. If its not, the reports will only be generated for users in the list.